doc/classes/Node3D.xml

```diff
@@ -224,143 +231,155 @@
+		<member name="global_position" type="Vector3" setter="set_global_position" getter="get_global_position" keywords="global_translation">
+			Global position (translation) of this node in global space (relative to the world). This is equivalent to the [member global_transform]'s [member Transform3D.origin].
 		</member>
 		<member name="global_rotation" type="Vector3" setter="set_global_rotation" getter="get_global_rotation">
-			Rotation part of the global transformation in radians, specified in terms of YXZ-Euler angles in the format (X angle, Y angle, Z angle).
-			[b]Note:[/b] In the mathematical sense, rotation is a matrix and not a vector. The three Euler angles, which are the three independent parameters of the Euler-angle parametrization of the rotation matrix, are stored in a [Vector3] data structure not because the rotation is a vector, but only because [Vector3] exists as a convenient data-structure to store 3 floating-point numbers. Therefore, applying affine operations on the rotation "vector" is not meaningful.
+			Global rotation of this node as [url=https://en.wikipedia.org/wiki/Euler_angles]Euler angles[/url], in radians and in global space (relative to the world). This value is obtained from [member global_basis]'s rotation.
+			- The [member Vector3.x] is the angle around the global X axis (pitch);
+			- The [member Vector3.y] is the angle around the global Y axis (yaw);
+			- The [member Vector3.z] is the angle around the global Z axis (roll).
+			[b]Note:[/b] Unlike [member rotation], this property always follows the YXZ convention ([constant EULER_ORDER_YXZ]).
 		</member>
 		<member name="global_rotation_degrees" type="Vector3" setter="set_global_rotation_degrees" getter="get_global_rotation_degrees">
-			Helper property to access [member global_rotation] in degrees instead of radians.
+			The [member global_rotation] of this node, in degrees instead of radians.
 		</member>
 		<member name="global_transform" type="Transform3D" setter="set_global_transform" getter="get_global_transform">
-			World3D space (global) [Transform3D] of this node.
+			The transformations of this node in global space (relative to the world). Contains and represents this node's [member global_position], [member global_rotation], and global scale.
+			[b]Note:[/b] Getting this property while the node is not inside the tree will fail and return [constant Transform3D.IDENTITY].
 		</member>
-		<member name="position" type="Vector3" setter="set_position" getter="get_position" default="Vector3(0, 0, 0)">
-			Local position or translation of this node relative to the parent. This is equivalent to [code]transform.origin[/code].
+		<member name="position" type="Vector3" setter="set_position" getter="get_position" default="Vector3(0, 0, 0)" keywords="translation">
+			Position (translation) of this node in local space (relative to the parent node). This is equivalent to the [member transform]'s [member Transform3D.origin].
```

Comment:
Node3D's position in its own local/object coordinate space is always zero (`Vector3.ZERO`). This `position` is in the parent coordinate system.

And so on for other _"in local space (relative to the parent node)"_...